src/main/java/org/opentripplanner/service/vehiclerental/model/VehicleTypeCount.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.service.vehiclerental.model;
+
+import java.io.Serializable;
+
+public class VehicleTypeCount implements Serializable, Comparable<VehicleTypeCount> {
```

Comment:
Since this is a very simple DTO, you can use a Java `record`. This will make this file tiny.